Makefile

```diff
@@ -93,6 +93,10 @@ format: deps
 docker:
 		docker build -f .docker/Dockerfile-build -t oryd/kratos:latest .
 
+# Runs the documentation tests
+test-docs:
+		text-run README.md --offline
+
```

Comment:
It wasn't directly obvious to me how to run this using your Circle config. I think either as the nancy test (but I don't know what that is) or via a separate Node Docker image. Any suggestions?